### PR TITLE
add workdir option to RunAs build step

### PR DESCRIPTION
**Pull request by popravich**

### CODE REVIEW

**Comment by tailhook** [comment]
1. We have `work-dir` (with a dash) in command, so should replicate it here
2. We allow relative dir (say `docs`) in command, so I don't see any reason why we would prohibit it here

**Comment by popravich** [comment]
done

File: src/builder/commands/generic.rs [hunk 1]
```diff
@@ -311,7 +315,11 @@ impl BuildStep for RunAs {
             setup_command(&guard.ctx, &mut cmd);
             cmd.arg("-exc");
             cmd.arg(&self.script);
-            cmd.current_dir("/work");
+            if self.work_dir.is_absolute() {
```
Comment (tailhook) [inline src/builder/commands/generic.rs]: You don't need this check, it's handled by `.join()` (https://is.gd/lIK4Fx)

**Comment by popravich** [comment]
fixed

**Comment by tailhook** [comment]
Merged. Thanks!

**Pull request merged**